src/store/repository/conversation.repository.ts

```diff
@@ -1,7 +1,11 @@
 import { decryptXChaCha20Poly1305, encryptXChaCha20Poly1305 } from "kaspa-wasm";
 import { DBNotFoundException, KasiaDB } from "./db";
 
-export type ConversationStatus = "pending" | "active" | "rejected";
+export type ConversationStatus =
+  | "sent-pending"
+  | "receive-pending"
+  | "active"
+  | "rejected";
```

Comment:
note: since we still don't have a deployed version of Index DB live, we don't have to migrate existing data